lib/kafka/consumer.rb

```diff
@@ -280,6 +280,21 @@ def each_batch(min_bytes: 1, max_wait_time: 5, automatically_mark_as_processed:
       end
     end
 
+    # @param min_bytes [Integer] the minimum number of bytes to read before
+    #   returning messages from the server; if `max_wait_time` is reached, this
+    #   is ignored.
+    # @param max_wait_time [Integer, Float] the maximum duration of time to wait before
+    #   returning messages from the server, in seconds.
+    # @param topics_offsets [Hash] the offsets to fetch from. By default it will
+    #   fetch from where the offset manager defaults too.
+    # @return batch [Kafka::FetchedBatch] a message batch fetched from Kafka.
+    def poll(min_bytes: 1, max_wait_time: 5, offsets: {})
```

Comment:
@dasch yeah i see what you mean. I think as a first implementation every consumer using the local seek could be enough. 

> Given the fact that consumer groups need to be relatively small anyway (more than 32 would be a stretch IMO)

i'm a little bit intrigued this? have you experienced instability or issues with groups larger than 32?